pkg/agent/handler/instance.go

```diff
@@ -26,7 +26,7 @@ func (h *instanceHandler) Handle(ctx context.Context, _ *proto.EventMeta, resour
 		return nil
 	}
 
-	if action == proto.Event_CREATED || action == proto.Event_UPDATED {
+	if action == proto.Event_CREATED || action == proto.Event_UPDATED || action == proto.Event_SUBRESOURCEUPDATED {
```

Comment:
The vapi-controller in Central creates the APIServiceInstance via a POST, followed quickly by a PUT to create the x-agent-details sub-resource. If we ignore the SUBRESOURCEUPDATED event it is possible that the agent SDK pulls the main resource into the cache before it has the x-agent-details subresource. Are there any impacts to adding this ? It seems to work OK.

Reply:
No impacts that I can think of